src/main/java/employeeServiceCRUD/service/AddressService.java

```diff
@@ -0,0 +1,38 @@
+package employeeServiceCRUD.service;
+
+import employeeServiceCRUD.models.Address;
+import employeeServiceCRUD.repositories.AddressRepositoryIml;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class AddressService {
+
+    AddressRepositoryIml addressRepositoryIml;
```

Comment:
here should be AddressRepository, not Iml. Iml you will set as value, but interface you use as a data type